logic app templates/audit-events-template.json

```diff
@@ -0,0 +1,181 @@
+                        "properties": {
+                          "tenantId": {
+                            "type": "string"
+                          },
+                          "timestamp": {
+                            "type": "string"
+                          }
+                        },
+                        "type": "object"
+                      },
+                      "name": {
+                        "type": "string"
+                      },
+                      "version": {
+                        "type": "string"
+                      }
+                    },
+                    "required": ["audit", "name", "version", "metadata"],
+                    "type": "object"
+                  },
+                  "type": "array"
+                }
+              }
+            },
+            "ParseEvent": {
+              "runAfter": {
+                "GetEvent": ["Succeeded"]
+              },
+              "type": "ParseJson",
+              "inputs": {
+                "content": "@outputs('GetEvent')",
+                "schema": {
+                  "properties": {
+                    "audit": {
+                      "properties": {
+                        "entityId": {
+                          "type": "string"
+                        },
+                        "entityName": {
+                          "type": "string"
+                        },
+                        "entityType": {
+                          "type": "string"
+                        },
+                        "operationName": {
+                          "type": "string"
+                        },
+                        "userId": {
+                          "type": "string"
+                        }
+                      },
+                      "type": "object"
+                    },
+                    "metadata": {
+                      "properties": {
+                        "tenantId": {
+                          "type": "string"
+                        },
+                        "timestamp": {
+                          "type": "string"
+                        }
+                      },
+                      "type": "object"
+                    },
+                    "name": {
+                      "type": "string"
+                    },
+                    "version": {
+                      "type": "string"
+                    }
+                  },
+                  "required": ["audit", "name", "version", "metadata"],
+                  "type": "object"
+                }
+              }
+            }
+          },
+          "outputs": {}
+        },
+        "parameters": {
+          "$connections": {
+            "value": {
+              "eventhubs": {
+                "connectionId": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Web/connections/eventhubs')]",
+                "connectionName": "eventhubs",
```

Comment:
When I tried running this template (via the "Deploy to Azure" button), I got an error pretty quickly:

![image](https://user-images.githubusercontent.com/165767/90285383-a7977300-de28-11ea-9c23-14d43d1f4efa.png)

I was never prompted to select an existing event hub (or create a new one).  Should I have been?

Reply:
This error appears to be because you don't have an Event Hub setup. I will include instructions regarding this prerequisite.